generators/entity-server/templates/src/main/java/package/domain/Entity.java.ejs

```diff
@@ -563,12 +564,12 @@ relationships.forEach((relationship, idx) => {
     <%_ if (relationshipType === 'one-to-many' || relationshipType === 'many-to-many') { _%>
 
     public Set<<%= asEntity(otherEntityNameCapitalized) %>> get<%= relationshipNameCapitalizedPlural %>() {
-        return <%= relationshipFieldNamePlural %>;
+        return this.<%= relationshipFieldNamePlural %>;
     }
         <%_ if (fluentMethods) { _%>
 
     public <%= asEntity(entityClass) %> <%= relationshipFieldNamePlural %>(Set<<%= asEntity(otherEntityNameCapitalized) %>> <%= otherEntityNamePlural %>) {
-        this.<%= relationshipFieldNamePlural %> = <%= otherEntityNamePlural %>;
```

Comment:
better to keep this ?

Reply:
forget my comment, I understand now :)

Reply:
Thanks. I was in a hurry, didn’t post the explanation.